app/lib/nomis_client/allocations.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module NomisClient
+  class Allocations < NomisClient::Base
+    class << self
+      def post(booking_id:, body_params: {})
+        allocations_path = "/bookings/#{booking_id}/prison-to-prison"
+
+        NomisClient::Base.post(allocations_path, body: body_params.to_json)
+      rescue OAuth2::Error => e
+        Raven.capture_message(
+          'Allocations::CreateInNomis Error!',
+          extra: {
+            allocations_route: allocations_path,
```

Comment:
Could we generalise this to path and have this exception method in the base client or something that makes other similarly propagated exceptions conform to the same schema?